packages/augur-ui/src/modules/alerts/actions/set-alert-text.ts

```diff
@@ -460,7 +459,7 @@ export default function setAlertText(alert: any, callback: any) {
       case GETORCACHEDESIGNATEDREPORTSTAKE:
         alert.title = "Get stake size required for desginated reports";
         break;
-      case GETORCACHEMARKETCREATIONCOST:
+      case getOrCacheValidityBond:
```

Comment:
Where is this case defined since GETORCACHEMARKETCREATIONCOST get removed? Also, should the title below be updated too?

Reply:
ah this should just be removed. GETORCACHEMARKETCREATIONCOST is gone in favor of just asking for the validity bond